.github/workflows/cd-release-linux.yaml

```diff
@@ -0,0 +1,87 @@
+name: Build dolt-workbench for Linux
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        description: GitHub Release Version Number
+        required: true
+        type: string
+
+  workflow_call:
+    inputs:
+      release-version:
+        description: Github Release Version Number
+        required: true
+        type: string
+
+jobs:
+  build-linux:
+    name: Build Linux release version
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Bump linux build version
+        uses: ./.github/actions/bump-build-version
+        with:
+          build-config-file: web/build/builder-linux-config.yaml
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "latest"
+
+      - name: Enable Corepack
+        run: corepack enable
+
+      - name: Cache Next.js build
+        uses: actions/cache@v4
+        with:
+          path: |
+            web/renderer/.next/cache
+            web/.next/cache
+          key: nextjs-${{ runner.os }}-${{ hashFiles('web/renderer/next.config.js', 'web/package.json', 'web/yarn.lock') }}
+          restore-keys: |
+            nextjs-${{ runner.os }}-
+
+      - name: Install graphql-server dependencies
+        working-directory: graphql-server
+        run: |
+          yarn 
+          yarn build
+
+      - name: Install web dependencies
+        working-directory: web
+        run: |
+          yarn
+          yarn download:dolt
```

Comment:
Doesn't need to be this PR but would be nice to be able to reuse these among the different build workflows